File: sol/1653/1653.py
class Solution:
    def minimumDeletions(self, s: str) -> int:
        res = 0
        b = 0
        for c in s:
            if c == 'b':
                b += 1
            elif b != 0:
                b -= 1
                res += 1
        return res

File: sol/1653/1653.cpp
struct Solution {
    inline int minimumDeletions(string &s) {
        int res = 0, b = 0;
        for (char c : s){
            if (c == 'b'){
                ++b;
            } else if (b != 0){
                --b;
                ++res;
            }
        }
        return res;
    }
};
